### PR TITLE
Update simple RAG use-case to support latest LangChain syntax

### PR DESCRIPTION
**Pull request by jacoblee93**
# What does this PR do?

Switches the legacy LangChain LLMChain to use expression language, which modernizes/removes some of the parsing.

## Who can review?

Feel free to tag members/contributors who may be interested in your PR.

@MKhalusova

### CODE REVIEW

File: notebooks/en/rag_zephyr_langchain.ipynb [hunk 1]
```diff
@@ -1,23 +1,10 @@
 {
```
Comment (MKhalusova) [inline notebooks/en/rag_zephyr_langchain.ipynb:1]: <p>Did you mean to add a code cell here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Comment by MKhalusova** [comment]
Thanks for improving the example! It looks like there's a code cell missing at the end though. Can you check?

**Comment by jacoblee93** [comment]
Oh, shoot - yeah I dropped an extra section and forgot to remove the lead-in. Will fix.

**Comment by jacoblee93** [comment]
Will manually cherry pick the change in the interim.

**Comment by jacoblee93** [comment]
Yeah the diff was odd - I readded `_Authored by: [Maria Khalusova](https://github.com/MKhalusova)_` which I think was the only thing?

**Comment by jacoblee93** [comment]
LMK if that looks better

**Comment by MKhalusova** [comment]
> Yeah the diff was odd - I readded `_Authored by: [Maria Khalusova](https://github.com/MKhalusova)_` which I think was the only thing?

Yes, that was the only thing added. Thanks for re-adding it!

**Review by MKhalusova — APPROVED** [review]
LGTM, thanks again!

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**